src/Simulations/time_step_wizard.jl

```diff
@@ -116,3 +116,15 @@ end
 (wizard::TimeStepWizard)(simulation) =
     simulation.Δt = new_time_step(simulation.Δt, wizard, simulation.model)
 
+"""
+    conjure_wizard!(simulation; interval=5, wizard_kw...)
+
+Add a `TimeStepWizard` built with `wizard_kw` as a `Callback` to `simulation`
+using the schedule `IterationInterval(interval)`.
+"""
+function conjure_wizard!(simulation; interval=5, wizard_kw...)
```

Comment:
Should we rename it to `conjure_adaptive_timestep_wizard`? I'm thinking that "wizard" is a bit esoteric since it involves knowing that we refer to adaptive time stepping as "timestep wizard". Is it?

Reply:
that would make the script easier to understand for someone reading it, but is a lot more boring :-(

Reply:
If we do make the name more verbose we should use `time_step_wizard` and omit `adaptive`

Reply:
Yes do that. More boring indeed, but less esoteric.